plugins/UM3NetworkPrinting/src/Cloud/CloudOutputDeviceManager.py

```diff
@@ -269,14 +276,14 @@ def _devicesRemovedFromAccount(self, removed_device_ids: Set[str]) -> None:
             return
 
         # Generate message
-        removed_printers_message = Message(
+        self.removed_printers_message = Message(
                 title = self.I18N_CATALOG.i18ncp(
                         "info:status",
                         "Cloud connection is not available for a printer",
                         "Cloud connection is not available for some printers",
                         len(self.reported_device_ids)
                 ),
-                lifetime = 0
+                lifetime = 30
```

Comment:
```suggestion

```
30 is the default, so I think it's better to leave it out altogether (my IDE feels the same way ;))